### ISSUE TITLE
Playlist Drag and Drop Sorting Improvements

### ISSUE THREAD

**Issue by deepspaceaxolotl** [FreeTubeApp/FreeTube#9467]
### Guidelines

- [x] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [x] I have encountered this bug in the [official downloads of FreeTube](https://github.com/FreeTubeApp/FreeTube#official-downloads). **The only official downloads are from the [https://freetubeapp.io](https://freetubeapp.io/#download) website, the [FreeTubeApp/FreeTube GitHub repository](https://github.com/FreeTubeApp/FreeTube/releases) and the [io.freetubeapp.FreeTube flatpak](https://flathub.org/en/apps/io.freetubeapp.FreeTube) from Flathub.**
- [x] I have [searched the issue tracker for **open and closed** issues](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+sort%3Arelevance-desc) that are similar to the bug report I want to file, without success.
- [x] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [x] This issue contains only one bug.
- [x] I have read and agree to follow the [rules](https://docs.freetubeapp.io/community/rules/).

### Describe the bug

Glad to see the drag and drop playlist sorting feature from #8681 now available! Having used it for a little bit now, I'm noticing a few spots that could be improved for better UX.

The main issue I'm finding is that dragging up or down is done at the same pace, not speeding up the further one moves one's mouse. This can make it take quite a while to move a video in long playlists, which can be both annoying and less accessible for those with issues holding down a mouse button uninterrupted for so long. I've also noticed that the page doesn't start scrolling up until the video is dragged quite far up over the playlist header – the viewport that gets taken into account seems to be the whole window, rather than just the part with the playlist grid (in grid mode).

The other issue is more of a visual downgrade, in that the addition of the area that can be used for dragging has created too much blank space between video rows in grid mode. Since that area only has the one central draggability indicator and is otherwise blank, it looks as though the blank space between video rows has been doubled.

### Expected Behavior

The speed of scrolling when dragging videos should be relative to how far up/down the mouse is, like in many other similar implementations, so that the time needed to drag it to the top/bottom of a playlist is not proportional to the length of the playlist. The point at which scrolling gets triggered should also take into account the playlist grid area in grid mode, rather than the whole window viewport (which includes the playlist header).

The blank space between video rows in grid mode should be removed/collapsed if the draggable space is there, so that the grid has a similar density as before.

### Issue Labels

accessibility issue, usability issue, visual bug

### FreeTube Version

v0.25.0 Beta

### Operating System Version

Windows 10, version 22H2

### Installation Method

.exe

### Primary API used

Local API

### Nightly Build

- [ ] I have encountered this bug in the latest **official** [nightly build](https://docs.freetubeapp.io/development/nightly-builds), do not check this box if you are using an unofficial or custom build.

**Comment by deepspaceaxolotl** [comment]
I think this should have been labelled an enhancement, my bad!

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Please create a feature request report for this
